mrt/forms/meetings/meeting.py

```diff
@@ -355,11 +355,12 @@ class MediaParticipantDummyForm(OrderedFieldsForm):
 
     class Meta:
         model = Participant
-        only = ('title', 'first_name', 'last_name', 'email', 'category_id')
+        only = ('title', 'first_name', 'last_name', 'email', 'category_id',
+            'verified')
```

Comment:
This line is under-indented. See https://www.python.org/dev/peps/pep-0008/#indentation  (_Aligned with opening delimiter_)